specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-05-01-preview/examples/metadata/DeleteMetadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2021-03-01-preview",
```

Comment:
Please update with new api version

Reply:
Sorry I think this comment was on one of the earlier commits.  The plan for this API was to add it to `2021-03-01-preview`, and they are all under that folder etc

Reply:
The APIs and the models defined in the swagger look good. But you need to release these new path additions in a new API version. This is a process enforced by the API breaking change review board. That is also the reason why breaking changes task is failing on this RP, See more here: http://aka.ms/bcforapi